pages/querying/clauses/alter.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: ALTER clause
```

Comment:
General comment: since we are introducing a new clause, it also needs to be listed at https://memgraph.com/docs/querying/clauses. I'll add it there as part of this PR.